ui/react-components/components/AppointmentEditor/AppointmentEditor.jsx

```diff
@@ -43,20 +44,31 @@ import {
     isSpecialitiesEnabled,
     maxAppointmentProvidersAllowed
 } from "../../helper.js";
+import {getSelectedWeekDays, getWeekDays} from "../../services/WeekDaysService/WeekDaysService";
+import ButtonGroup from "../ButtonGroup/ButtonGroup.jsx";
+import {getErrorTranslations} from "../../translations/errorTranslations";
 
 const AppointmentEditor = props => {
+
+    const {appConfig, intl} = props;
+    const {angularState} = React.useContext(AppContext);
```

Comment:
What is angularState???

Reply:
We need to set view date to navigate to the appointment booked date. so we got the state separately from angular to react